resources/assets/wailaharvestability/lang/en_US.lang

```diff
@@ -30,3 +30,9 @@ option.harvestability.silktouchability=Silk Touchability
 option.harvestability.silktouchability.sneakingonly=Sneak only: Silk Touchability
 option.harvestability.harvestlevelnum=Harvest level no.
 option.harvestability.harvestlevelnum.sneakingonly=Sneak only: Harvest level no.
+
+# harvestlevels
+wailaharvestability.harvestlevel0=Wood
+wailaharvestability.harvestlevel1=Stone
+wailaharvestability.harvestlevel2=Iron
+wailaharvestability.harvestlevel3=Diamond
```

Comment:
seems odd to me, 
default its named after vanilla; and if a mod/modpack changes that, they should change the translation then imo

but will change it anyways